src/engine/client/cg_msgdef.h

```diff
@@ -313,6 +314,10 @@ namespace Render {
 		IPC::Message<IPC::Id<VM::QVM, CG_R_GETSHADERNAMEFROMHANDLE>, int>,
 		IPC::Reply<std::string>
 	>;
+	using GetEngineTimersMsg = IPC::SyncMessage<
+		IPC::Message<IPC::Id<VM::QVM, CG_R_GETENGINETIMERS>>,
+		IPC::Reply<EngineTimers*>
```

Comment:
Sending a pointer through inter-process communication doesn't make sense.

Reply:
Yeah, I think it can just be made shared instead.